cnf-tests/testsuites/e2esuite/dpdk/dpdk.go

```diff
@@ -50,10 +50,16 @@ import (
 )
 
 const (
-	LOG_ENTRY              = "Accumulated forward statistics for all ports"
-	DEMO_APP_NAMESPACE     = "dpdk"
-	SERVER_TESTPMD_COMMAND = "testpmd -l ${CPU} -w ${PCIDEVICE_OPENSHIFT_IO_%s} --iova-mode=va -- -i --portmask=0x1 --nb-cores=2 --forward-mode=mac --port-topology=loop --no-mlockall"
-	CLIENT_TESTPMD_COMMAND = "testpmd -l ${CPU} -w ${PCIDEVICE_OPENSHIFT_IO_%s} --iova-mode=va -- -i --portmask=0x1 --nb-cores=2 --eth-peer=0,ff:ff:ff:ff:ff:ff --forward-mode=txonly --no-mlockall"
+	LOG_ENTRY                 = "Accumulated forward statistics for all ports"
+	DEMO_APP_NAMESPACE        = "dpdk"
+	SERVER_TESTPMD_COMMAND    = "testpmd -l ${CPU} -w ${PCIDEVICE_OPENSHIFT_IO_%s} --iova-mode=va -- -i --portmask=0x1 --nb-cores=2 --forward-mode=mac --port-topology=loop --no-mlockall"
+	CLIENT_TESTPMD_COMMAND    = "testpmd -l ${CPU} -w ${PCIDEVICE_OPENSHIFT_IO_%s} --iova-mode=va -- -i --portmask=0x1 --nb-cores=2 --eth-peer=0,ff:ff:ff:ff:ff:ff --forward-mode=txonly --no-mlockall"
+	CREATE_TAP_DEVICE_COMMAND = `
+		mkdir -p /dev/net
+		mknod /dev/net/tun c 10 200
+		chmod 600 /dev/net/tun
```

Comment:
there is no need to create the device any more. just leave the `ip tuntap` command

Reply:
Done